modules/objdetect/src/precomp.hpp

```diff
@@ -50,6 +50,7 @@
 #include "opencv2/core/utility.hpp"
 #include "opencv2/core/ocl.hpp"
 #include "opencv2/core/private.hpp"
+#include "opencv2/core/utils/logger.hpp"
```

Comment:
Logger is usually configured per .cpp file. Placing it here blocks that. No need to put it into precomp.hpp